README.MD

```diff
@@ -1,32 +1,48 @@
 # Schuldenteller.nl
+
 Source code of [schuldenteller.nl](https://schuldenteller.nl).
 
-# NOTICE
-This project is still a work in progress.
+> ⚠️ This project is still a work in progress.
 
+<center>
+    <img src="https://user-images.githubusercontent.com/16303480/101277469-f345a200-37b4-11eb-96dc-82b0b5de222d.png" alt="Screenshot" />
+</center>
 
-## Local development
+## Deploying
 
-After cloning:
+Make sure you have at least an [LTS version of NodeJS](https://nodejs.org/) installed.
 
 ```bash
+git clone git@github.com:NietMijnSchuld/schuldenteller.git
+cd schuldenteller
 yarn install
+yarn prod
+```
 
-# Build tailwind locally
-yarn tailwind
+You now have an optimised build of the website suitable for a production environment.
+You may serve the `/public` directory as a static site.
 
-# Compile typescript
-yarn tsc --watch
-```
+## Local development
 
-## Build for production
+After cloning:
 
 ```bash
 yarn install
 
-# Build tailwind and run PurgeCSS
-NODE_ENV=production yarn tailwind
+# Builds dev-build of the CSS, compiles TypeScript and watches .ts files.
+yarn dev
+```
+
```

Comment:
Perhaps it would be nice to add here that the webserver should be configured manually, and should simply serve the `public` folder?

Reply:
Good suggestion, will fix.